### ISSUE TITLE
bundle install using chef bundle with chef's ruby, not the one intended

### ISSUE THREAD

**Issue by phongsi** [ajgon/opsworks_ruby#31]
I was trying to install rack but failed due to ruby versions being old. 
```
Gem::InstallError: rack requires Ruby version >= 2.2.2
```

That didn't make sense because the cookbook installs ruby 2.3 by default.
https://github.com/ajgon/opsworks_ruby/blob/master/attributes/default.rb#L18
and the bundler should be pointing to that ruby
https://github.com/ajgon/opsworks_ruby/blob/master/recipes/setup.rb#L26
```
[deploy@ip-10-0-10-198 ~]$ which bundle
/usr/local/bin/bundle
```

But in reality the bundle that runs on bundle install is
```
/opt/chef/embedded/bin/bundle
```
it wasn't a problem when this cookbook used https://github.com/poise/application_ruby
but i do agree with the change.

**Comment by phongsi** [comment]
btw @ajgon i'm working on a fix for this

**Comment by ajgon** [comment]
Resolved in #32
